app/controllers/spree/admin/gift_cards_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
 
       private
       def collection
-        Spree::GiftCard.order("created_at desc").page(params[:page]).per(Spree::Config[:orders_per_page])
+        Spree::GiftCard.order(created_at: :desc).page(params[:page]).per(Spree::Config[:orders_per_page])
```

Comment:
@sawangupta92 - I think we can use collection method of ResourceController here.